_static/css/custom.css

```diff
@@ -0,0 +1,15 @@
+.wy-side-nav-search {
+	background-color: #dd1144;
+}
+
+dl.post, dl.patch, dl.delete, dl.get, dl.put {
+	display: none
+}
+
+[sticky-sidebar][_ngcontent-vyl-1], .menu-content {
+	max-height: calc(100% - 200px) !important;
```

Comment:
Do we have any limitations regarding supported browsers? This is quite new syntax.

Reply:
Added from CSS that are used in docs